nilearn/signal.py

```diff
@@ -462,18 +493,24 @@ def clean(signals, sessions=None, detrend=True, standardize=True,
                 session_confounds = confounds[sessions == s]
             signals[sessions == s, :] = \
                 clean(signals[sessions == s],
-                      detrend=detrend, standardize=standardize,
+                      detrend=detrend, normalize=normalize,
                       confounds=session_confounds, low_pass=low_pass,
                       high_pass=high_pass, t_r=2.5)
 
-    # detrend
+    if normalize != 'psc':
+        if standardize:
+            normalize = 'std'
+        else:
+            normalize = None
```

Comment:
I'd rather have an error raised if normalize is no in {'psc', 'std', or None}. That way typos lead to errors, and are not swallowed silently.